spec/controllers/webauth_controller_spec.rb

```diff
@@ -30,6 +30,22 @@
     end
   end
 
+  describe '#login_media_download' do
+    subject { get :login_media_download, id: 'bb582xs1304', file_name: 'bb582xs1304_sl', format: 'mp4' }
+
+    it 'returns the user to the media download action' do
+      expect(subject).to redirect_to media_url(controller.params.symbolize_keys)
+    end
+  end
+
+  describe '#login_media_stream' do
+    subject { get :login_media_stream, id: 'bb582xs1304', file_name: 'bb582xs1304_sl.mp4', format: 'm3u8' }
```

Comment:
totally possible that i'm missing something here, but would `format: 'mp4'` be more reasonable here?  (since it seems like `m3u8` isn't a media format).